layer/decorators/assertions_decorator.py

```diff
@@ -344,60 +382,3 @@ def assert_func(result: Any) -> Any:
         return result
 
     return assert_func
-
-
-def get_asserted_function(function: Callable[..., Any]) -> Callable[..., Any]:
-    asserted_function = function
-    assertion_list = _get_assertion_list(function)
-    for assertion in assertion_list:
-        name = assertion[0]
-        values = assertion[1:]
-        assertion_function = _assertion_name_function_mapper(name, values)
-        asserted_function = _call_function_with_assertion(
-            asserted_function, assertion_function
-        )
-
-    return asserted_function
-
-
-def get_assertion_functions_data(function: Callable[..., Any]) -> List[Assertion]:
```

Comment:
This is still being used in `monorepo`, why are you deleting it?

Reply:
I've moved this to https://github.com/layerai/sdk/pull/137, but this is basically no longer necessary - we can just use `func.layer.get_assertions()` to get the same result without a helper function.